docs/README.md

```diff
@@ -3,32 +3,32 @@
 
 Using gRPC provides a number of benefits:
-- The SSE plugin (or analytic connection) can be deployed on a computer other than the one Qlik Sense is deployed on.
+- The SSE plugin (or analytic connection) can be deployed on a computer other than the one Qlik is deployed on.
 - You can easily create SSE plugins in any of the languages that gRPC supports out of the box (over ten languages).
 
 The SSE protocol also makes use of [protocol buffers](https://developers.google.com/protocol-buffers/), Google's language-neutral, platform-neutral, extensible mechanism for serializing structured data. We have specified all default values according to [Optional Fields And Default Values](https://developers.google.com/protocol-buffers/docs/proto#optional).
 
-The following diagram shows an example of SSE plugins communicating with Qlik Sense:
+The following diagram shows an example of SSE plugins communicating with Qlik:
 
 ![](SSE_overview.png)
 
 **Note:** In the context of SSE communication:  
-- Qlik Sense is the **client**
+- Qlik is the **client**
 - The SSE plugin (Python, etc.) is the **server**.  
 
-This means that the server (that is, the plugin) must be started and running before Qlik Sense is started.
+This means that the server (that is, the plugin) must be started and running before the Qlik engine is started.
 
 ## Considerations
 * **Potential bottleneck**  
 SSE plugins will feel slower than the normal Qlik engine calculations that use in-memory data. SSE plugins may also become a bottleneck if massive usage is applied (many measures, many users, many apps, a lot of data), so be sure to architect your SSE deployment very carefully.
 
-* **Order of rows (data) sent from Sense**  
-The order of the data sent to the SSE plugin may not be the same as the order you see in the visualization. It is up to you as a developer to preserve the order when you return the values from the plugin to the Qlik Sense client.
+* **Order of rows (data) sent from Qlik**  
+The order of the data sent to the SSE plugin may not be the same as the order you see in the visualization. It is up to you as a developer to preserve the order when you return the values from the plugin to the Qlik client.
 
 * **Recovering lost connection**  
-If the connection to the plugin is lost, Qlik Sense attempts to reconnect after 20 seconds by default. You can configure a different timeout interval in QMC.
+If the connection to the plugin is lost, the Qlik engine attempts to reconnect after 20 seconds by default. You can configure a different timeout interval in QMC.
 
 * **New log file**  
-You will see a new type of log file in the Qlik Sense engine logs containing SSE-specific information.
+You will see a new type of log file in the Qlik Sense engine logs and the QlikView Server logs containing SSE-specific information.
```

Comment:
I was thinking if we should provide more information about SSE logs in general, e.g. where to find the specific log files, how to change verbosity etc? Or is it enough to have it as is? Also, there are logs for QlikView Desktop as well. One must enable the ExtendedErrorLoggingMode setting and the information is added to the QlikViewError.log file and not to a separate log file as in the other cases. I know the logs in QlikView are not usually used by the users unless there are any problems, but it might be a good idea to mention in this case? since the development of apps using SSE will be done mainly in Desktop, if I've understood it correctly?

Reply:
I think it should be removed from here and only documented on help.qlik.com. It is a Qlik product specific feature.

Reply:
That's true, good point.

Reply:
I removed the section.